src/main/java/org/broadinstitute/dsm/model/participant/ParticipantWrapper.java

```diff
@@ -0,0 +1,259 @@
+import org.broadinstitute.dsm.db.dao.ddp.instance.DDPInstanceDao;
+import org.broadinstitute.dsm.db.dao.ddp.participant.ParticipantDataDao;
+import org.broadinstitute.dsm.db.dto.ddp.instance.DDPInstanceDto;
+import org.broadinstitute.dsm.db.dto.ddp.participant.ParticipantDataDto;
+import org.broadinstitute.dsm.model.elasticsearch.ESProfile;
+import org.broadinstitute.dsm.model.elasticsearch.ElasticSearch;
+import org.broadinstitute.dsm.model.elasticsearch.ElasticSearchParticipantDto;
+import org.broadinstitute.dsm.model.elasticsearch.ElasticSearchable;
+import org.broadinstitute.dsm.model.participant.data.FamilyMemberConstants;
+import org.broadinstitute.dsm.model.at.DefaultValues;
+import org.broadinstitute.dsm.statics.DBConstants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Data
+public class ParticipantWrapper {
+
+    private static final Logger logger = LoggerFactory.getLogger(ParticipantWrapper.class);
+
+    public static final String BY_DDP_PARTICIPANT_ID_IN = " AND request.ddp_participant_id IN (\"";
+    public static final String ORDER_AND_LIMIT = " ORDER BY request.dsm_kit_request_id desc LIMIT 5000";
+    public static final int DEFAULT_ROWS_ON_PAGE = 50;
+
+    private ParticipantWrapperPayload participantWrapperPayload;
+    private ElasticSearchable elasticSearchable;
+
+    private ElasticSearch esData = new ElasticSearch();
+    private List<Participant> participants = new ArrayList<>();
+    private Map<String, List<MedicalRecord>> medicalRecords = new HashMap<>();
+    private Map<String, List<OncHistoryDetail>> oncHistoryDetails = new HashMap<>();
+    private Map<String, List<KitRequestShipping>> kitRequests = new HashMap<>();
+    private Map<String, List<AbstractionActivity>> abstractionActivities = new HashMap<>();
+    private Map<String, List<AbstractionGroup>> abstractionSummary = new HashMap<>();
+    private Map<String, List<ElasticSearchParticipantDto>> proxiesByParticipantIds = new HashMap<>();
+    private Map<String, List<ParticipantDataDto>> participantData = new HashMap<>();
+
+    public ParticipantWrapper(ParticipantWrapperPayload participantWrapperPayload, ElasticSearchable elasticSearchable) {
+        this.participantWrapperPayload = Objects.requireNonNull(participantWrapperPayload);
+        this.elasticSearchable = Objects.requireNonNull(elasticSearchable);
+    }
+
+    public ParticipantWrapperResult getFilteredList() {
+        logger.info("Getting list of participant information");
+
+        DDPInstanceDto ddpInstanceDto = participantWrapperPayload.getDdpInstanceDto()
+                .orElseThrow();
+
+        if (StringUtils.isBlank(ddpInstanceDto.getEsParticipantIndex())) {
+            throw new RuntimeException("No participant index setup in ddp_instance table for " + ddpInstanceDto.getInstanceName());
+        }
+
+        DDPInstance ddpInstance = DDPInstance.getDDPInstance(ddpInstanceDto.getInstanceName());
```

Comment:
we need to change that to `        DDPInstance ddpInstance = DDPInstance.getDDPInstanceWithRole(ddpInstanceDto.getInstanceName(), DBConstants.HAS_MEDICAL_RECORD_ENDPOINTS);
` to have the role in L135 and L138 and to fix https://broadinstitute.atlassian.net/browse/DDP-6809